implementations/rust/ockam/ockam_command/src/identity/show.rs

```diff
@@ -1,18 +1,17 @@
-use crate::node::NodeOpts;
 use crate::util::output::Output;
-use crate::util::{extract_address_value, node_rpc, Rpc};
-use crate::CommandGlobalOpts;
+use crate::util::node_rpc;
+use crate::{CommandGlobalOpts, OutputFormat};
 use clap::Args;
 use core::fmt::Write;
 use ockam::Context;
 use ockam_api::nodes::models::identity::{LongIdentityResponse, ShortIdentityResponse};
-use ockam_core::api::Request;
 use ockam_identity::change_history::IdentityChangeHistory;
+use ockam_identity::Identity;
+use ockam_vault::Vault;
 
 #[derive(Clone, Debug, Args)]
 pub struct ShowCommand {
-    #[command(flatten)]
-    node_opts: NodeOpts,
+    name: String,
```

Comment:
Something I forgot to describe on the issue is that this argument must be an `Option<String>`, and default to whatever the default identity is. Check out this to learn how to do it: https://github.com/build-trust/ockam/issues/3931

Reply:
Also, note that we already merged a [PR](https://github.com/build-trust/ockam/pull/3938) for this command. Still, this PR can be a good chance to add the improvement I mentioned in the previous comment.